### DISCUSSION TITLE
detection playbook from filebeat

### DISCUSSION THREAD

**Discussion by dubnik** [Security-Onion-Solutions/securityonion#7791] [Unsupported Versions] [answered]
Hi

I am sending log from fortigate logs via filebeat to elk. I have logs stored  in elk.

Is it possible to create detection playbook from elk logs in SO.

Thank you

**Comment by defensivedepth** [comment, marked as answer]
Yes, any log that is in Elasticsearch you can write a Sigma rule in Playbook for.

You can see an example here: https://www.youtube.com/watch?v=IS2SOlDedPc&list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH&index=7

**Reply by dubnik** [reply]
thanks i have read and saw videos about creating playbook, but I haven't find any materials related to create sigma rule from custom filebeat/logstash logs stored in elk. What should be used as logsource: product

Thanks
regards

**Reply by defensivedepth** [reply]
@dubnik What type of custom logs are they?
